app/models/test.rb

```diff
@@ -6,5 +6,17 @@ class Test < ActiveRecord::Base
 
   accepts_nested_attributes_for :answers
 
+  after_create :count_result
+
   resourcify#rolify
+
+  class_attribute :current_user
```

Comment:
because I use variable current_user in model here https://github.com/begin29/schoolTest/pull/19#discussion-diff-51063448R16